src/main/java/UnknownCommandException.java

```diff
@@ -0,0 +1,5 @@
+public class UnknownCommandException extends Exception {
```

Comment:
Perhaps add more Javadoc comments to explain more about the functionality of your code or the purpose of creating certain classes?